templates/web/base/report/new/category.html

```diff
@@ -1,4 +1,22 @@
-[% IF category_options.size ~%]
+[% IF category_groups.size > 1 ~%]
+    [% IF category;
+        category_lc = category | lower;
+       END; ~%]
+    <label for='form_category' id="form_category_label">
+        [%~ loc('Category') ~%]
+    </label>[% =%]
+    <select class="form-control" name='category' id='form_category' data-role='[% c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
```

Comment:
This no longer matches the thing it's copying below. Could this be refactored so it's sharing all the bits that are shared?